packages/utils/src/libp2p/addressSorters.ts

```diff
@@ -31,6 +31,20 @@ export function isMultiaddrLocal(multiaddr: Multiaddr): boolean {
   }
 }
 
+export function getIpv4LocalAddressClass(address: Multiaddr): 'A' | 'B' | 'C' | 'D' | undefined {
+  if (isMultiaddrLocal(address)) {
+    if (address.toString().startsWith('/ip4/10.')) return 'A'
+
+    if (address.toString().startsWith('/ip4/172.16')) return 'B'
```

Comment:
This is not correct. The local IP address range goes from 172.16.0.0 – 172.31.255.255, see https://en.wikipedia.org/wiki/IPv4#Private_networks

Reply:
I noticed that already and fixed that :)